test/Graphics/Shape.cpp

```diff
@@ -99,4 +100,12 @@ TEST_CASE("sf::Shape class - [graphics]")
         CHECK(triangleShape.getGlobalBounds().width == Approx(3.f));
         CHECK(triangleShape.getGlobalBounds().height == Approx(2.f));
     }
+
+    SUBCASE("Copy and move semantics")
+    {
+        static_assert(std::is_copy_constructible<TriangleShape>::value);
+        static_assert(std::is_copy_assignable<TriangleShape>::value);
+        static_assert(std::is_move_constructible<TriangleShape>::value);
+        static_assert(std::is_move_assignable<TriangleShape>::value);
+    }
```

Comment:
Why is `TriangleShape` being tested and not `Shape`?

Reply:
Good catch. To be consistent with other non-constructible types, I need to directly test `sf::Shape`.